vscode/webviews/Chat.tsx

```diff
@@ -261,7 +261,7 @@ const TextArea: React.FunctionComponent<ChatUITextAreaProps> = ({
     chatModels,
 }) => {
     const inputRef = useRef<HTMLTextAreaElement>(null)
-    const placeholder = 'Message (@ to include code, / for commands)'
+    const placeholder = 'Message (@ to include files, @# for symbols, or / for commands)'
```

Comment:
I like the idea of updating this! I'd suggest one of these two versions though:

```suggestion
const placeholder = 'Message (@ to include files and symbols, or / for commands)'
```

or

```suggestion
const placeholder = 'Message (@ to include a file or symbol, or / for commands)'
```

I think the latter would be my preference.

Reply:
I think we need the `@#` instruction here because otherwise people don't know how to include a symbol. That's what I was getting at in the description: 

> For one, I initially was trying to just type # to include a symbol (the current language doesn't make it clear that you actually need @#), but this is also something we've heard [internally](https://sourcegraph.slack.com/archives/C04MZPE4JKD/p1705996369006999?thread_ts=1705105369.141479&cid=C04MZPE4JKD) and from a user DM in Discord.